examples/run-blockchain/index.js

```diff
@@ -44,9 +54,9 @@ async.series([
 
   // create and add genesis block
   function (next) {
-    genesisBlock.header = new BlockHeader(
-                            testData.genesisBlockHeader
-                          )
+    genesisBlock.header = new BlockHeader(formatBlockHeader(testData.genesisBlockHeader),
```

Comment:
I think this new `formatBlockHeader ` isn't needed. I just tried it without it and it works fine. Can we remove it?

Reply:
OK, I'll remove it.

Reply:
done